pkg/alert/alert.go

```diff
@@ -142,9 +143,7 @@ func (q *Queue) Len() int {
 
 // Cap returns the fixed capacity of the queue.
 func (q *Queue) Cap() int {
-	q.mtx.Lock()
-	defer q.mtx.Unlock()
-	return cap(q.queue)
+	return q.capacity
```

Comment:
not concurrent safe getter is fine?

Reply:
Also it is kind of static right? We don't want to change it or something?

Reply:
Yea, it's a constant passed into the constructor. We never modify it.
Not sure whether it's a global constant though. It could be fine... OTOH though in really big setups, especially if there's one global rule layer, a really high amount of alerts are very possible and we might want to make add a flag for it.